assets/show-more.js

```diff
@@ -0,0 +1,26 @@
+class ShowMoreButton extends HTMLElement {
+  constructor() {
+    super();
+    this.addEventListener('click', (event) => {
+      if (event.target.nodeName === 'SPAN' && event.target.closest('[id^="Show-More-"]')) {
+        this.expandShowMore(event.target.closest('[id^="Show-More-"]'));
+      }
+      if (event.target.nodeName === 'BUTTON' && event.target.closest('[id^="Show-More-"]')) {
+        this.expandShowMore(event.target.closest('[id^="Show-More-"]'));
+        const nextElementToFocus = event.target.parentNode.parentNode.querySelector(".facets__item-show-more")
+        if (!nextElementToFocus.classList.contains('.hidden')) {
+          nextElementToFocus.querySelector('input').focus()
+        }
+      }
+    });
+  }
+  expandShowMore(showMore) {
+    const facetsDiplay = showMore.closest('.facets__display-vertical');
```

Comment:
Again, if we've decided to split out show-more as a standalone component, I don't think its JS should refer to facets. This needs some work to be more generalized.